packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -446,6 +452,10 @@
             padding-left: 16px;
             padding-right: 16px;
         }
+        @include tablet-screen {
```

Comment:
should't we use padding-inline-start and padding-inline-end to support rtl as well?
cc: @suisin-deriv

Reply:
Done

Reply:
Done